doc/source/user/advanced.txt

```diff
@@ -5,6 +5,33 @@ Advanced Usage
 Writing a custom component (TODO)
 ---------------------------------
 
+Debugging batou runs
```

Comment:
Thanks for getting this documented. I'd like to see a few more details:

* how do you exactly configure the environment variables? is it sufficient to call batou locally with those or do we have to do extra work to get them to the remote side? even if nothing special needs to happen, I'd love an explicit example. including an example shell session how this looks like.
* how do you call the debugger on Python 3.6?
* any chance to cover this with some testing?

Reply:
* Environment variables are currently not propagated to the remote host. I do not know how to do this.
* I added an example debugging session to the documentation.
* I documented how to use the debugger in Python 3.6.
* I added a simple test to check the default configuration. Testing the usage of the debugger I see no chance.